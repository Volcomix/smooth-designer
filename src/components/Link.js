//@flow
import React from 'react'
import './Link.css'

export type Props = {
  fromX: number,
  fromY: number,
  toX: number,
  toY: number,
}

const Link = ({ fromX, fromY, toX, toY }: Props) => (
  <svg className="Link">
    <line className="Link-line" x1={fromX} y1={fromY} x2={toX} y2={toY} />
  </svg>
)

export default Link
