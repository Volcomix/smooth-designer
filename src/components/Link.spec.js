//@flow
import React from 'react'
import { shallow } from 'enzyme'
import Link from './Link'

it('renders without crashing', () => {
  const props = { fromX: 10, fromY: 20, toX: 30, toY: 40 }
  const wrapper = shallow(<Link {...props} />)
  expect(wrapper).toMatchSnapshot()
})
