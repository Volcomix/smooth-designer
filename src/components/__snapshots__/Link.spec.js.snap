// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<svg
  className="Link"
>
  <line
    className="Link-line"
    x1={10}
    x2={30}
    y1={20}
    y2={40}
  />
</svg>
`;
